admin/acceptance.py

```diff
@@ -946,7 +946,7 @@ def main(reactor, args, base_path, top_level):
                 parallel([
                     run_remotely(
                         username='root',
-                        address=node.address,
+                        address=node.address.encode('ascii'),
```

Comment:
`run_remotely` creates a `RunRemotely`.  `RunRemotely.address` is already constrained to be of type `bytes`.  So `node.address` must be of type `bytes` (on CPython) or this code wouldn't have worked without this change.

So it seems more likely that the problem is that `node.address` varies between `bytes` and `unicode` as a result of a bug somewhere else.

`bytes.encode('ascii')` is allowed by Python but it's a wrong thing to do and I think that's what this change is causing to happen on CPython.